packages/projects/src/components/ProjectForm.jsx

```diff
@@ -296,7 +312,47 @@ const ProjectFormAlert = ({ formStatus, errorMessage, projectHandle }) => {
 const ProjectForm = ({ mode, projectHandle }) => {
   const [state, dispatch] = useReducer(reducer, INITIAL_STATE)
 
-  const { services } = useServices()
+  const { services, currentUser } = useServices()
+  const user = currentUser()
+
+  useEffect(() => {
+    (async () => {
+      const { iss, groupIds } = user
+
+      const { entityFactory } = await services()
+      const groupsApi = await entityFactory({ resource: 'group' })
+      const userGroups = await Promise.all(groupIds.map(id => groupsApi.get(id)))
+
+      const membersApi = await entityFactory({ resource: 'member' })
+      const member = await membersApi.get(iss)
+
+      let profileName
```

Comment:
The user object already has a nickname where as not every user might have a profile.

Why are we not defaulting to the nickname instead? Also doesn't require another call to the backend.

Reply:
Because I was considering the nickname to be private. If a user elects not to share their profile then it would seem that they want to be anonymous. But if their adventure has their nickname attached to it, that would give them away, and it gives them no option to hide their name from the adventure.

If their nickname is tied to the adventure, I could hide it from being rendered in other views if the associated user doesn't have or doesn't share their profile. But then that requires the extra API calls in other places, that we wanted to cut down on. Plus it would still be visible in the HTTP requests.

And secondarily for people who DO have and share their profile, it would be confusing and inconsistent to see their wallet nickname tied to their adventure, when from a product perspective it should rightly be their profile name that serves as their display name everywhere.